src/gdcdictionary/schemas/exposure.yaml

```diff
@@ -127,6 +127,7 @@ properties:
       - Heavy Drinker
       - Lifelong Non-Drinker
       - Non-Drinker
+      - Not Heavy Drinker
```

Comment:
IMO: these categories would be more meaningful if it described units of alcohol per week.